t/25-cache.t

```diff
@@ -78,8 +78,8 @@ END { stop_service($server_instance) }
 
 my $app = OpenQA::CacheService->new(log => $log);
 my $cache = $app->cache;
-is $cache->sqlite->migrations->latest, 4, 'version 4 is the latest version';
-is $cache->sqlite->migrations->active, 4, 'version 4 is the active version';
+is $cache->sqlite->migrations->latest, 5, 'version 4 is the latest version';
+is $cache->sqlite->migrations->active, 5, 'version 4 is the active version';
```

Comment:
Now the description strings are wrong. That's the reason why the description string should not even contain the version number. Better:

```suggestion
is $cache->sqlite->migrations->latest, 5, 'Current version is the latest version';
is $cache->sqlite->migrations->active, 5, 'Current version is the active version';
```